.devcontainer/devcontainer.json

```diff
@@ -11,12 +11,13 @@
     "mounts": [
         "source=freqtrade-bashhistory,target=/home/ftuser/commandhistory,type=volume"
     ],
+    "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/freqtrade,type=bind,consistency=cached",
```

Comment:
is this actually necessary? 
I've tried to create a devcontainer config in vscode on an (almost) empty directory - and the only keys required were "name", "image" and "features" (which we apparently don't use - as we use the freqtrade upstream docker image with requirements-dev installed).

Reply:
I remember I could not find the workspaces folder without it. I will check again

Reply:
It actually does not work without this binding. So it has to stay

Reply:
i'm not sure about that. The devcontainer is also supposed to work on github codespaces - which it does in it's current form just fine (without any modification).

i still have to try the local version - but last time i tried, it worked just fine.